index.js

```diff
@@ -161,6 +161,7 @@ function fetchAllCommentsAfterNode(comments, nextNode, node) {
 
 module.exports = postcss.plugin('postcss-sorting', function (opts) {
     var linesBetweenChildrenRules = getLinesBetweenChildrenFromOptions(opts);
+    var smartGroupIndentation = !!(opts || {})['smart-group-indent'] || false;
```

Comment:
Not explicit type conversion. Should be more straightforward and clear. Complex one-liners are not good for maintainability.